common/changes/@rushstack/heft/typescript-solution_2021-08-11-21-38.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Remove copyFromCacheMode in typescript.json",
```

Comment:
Won't we need to update the cyclic dependency on Heft before this will work?

Reply:
Hm, good question.

Reply:
No projects are actually setting this to a value in typescript.json; it is commented out everywhere.